src/api/pipelineRuns.js

```diff
@@ -92,20 +93,20 @@ export function createPipelineRun({
     apiVersion: 'tekton.dev/v1beta1',
     kind: 'PipelineRun',
     metadata: {
-      name: `${pipelineName}-run-${Date.now()}`,
+      name,
       labels
     },
     spec: {
       pipelineRef: {
         name: pipelineName
       },
-      resources: Object.keys(resources).map(name => ({
-        name,
-        resourceRef: { name: resources[name] }
+      resources: Object.keys(resources).map(resourceName => ({
+        resourceName,
```

Comment:
This field should be called `name` to match the expected key in the spec. Unit tests related to this functionality are failing.

```suggestion
        name: resourceName,
```